src/app/modules/submission/encrypt-submission/IncomingEncryptSubmission.class.ts

```diff
@@ -60,9 +60,12 @@ export default class IncomingEncryptSubmission extends IncomingSubmission {
 
   responseVisibilityPredicate(response: FieldResponse): boolean {
     return (
-      'answer' in response &&
-      typeof response.answer === 'string' &&
-      response.answer.trim() !== ''
+      ('answer' in response &&
+        typeof response.answer === 'string' &&
+        response.answer.trim() !== '') ||
+      ('answerArray' in response &&
+        typeof response.answerArray === 'object' &&
```

Comment:
Some gotchas here :sweat
- in nodejs, `object`` is anything that is not undefined, boolean, number, bigint, string, symbol, or fn.
- null is also type `object`
  ```
  const x = null
  typeof x // 'object'
  ```
  
- You might want to use `Array.isArray()` instead

Reply:
Thanks for pointing this out! Done in [6b16843](https://github.com/opengovsg/FormSG/pull/6665/commits/6b16843491fc0330a551d23e5d9cdc50431f7a48).